crates/aptos/src/account/key_rotation.rs

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::common::types::{
```

Comment:
do we also need to support getting the original address ?

Reply:
yes. I think we should. will add that command